modules/javafx.base/src/test/java/test/javafx/util/converter/DateTimeStringConverterTest.java

```diff
@@ -62,127 +63,90 @@ public class DateTimeStringConverterTest {
         VALID_DATE_WITHOUT_SECONDS = c.getTime();
     }
 
-    private static Collection implementations() {
-        return Arrays.asList(new Object[][] {
-                { new DateTimeStringConverter(),
+    private record TestCase(DateTimeStringConverter converter, Locale locale, int dateStyle, int timetyle, String pattern,
+            DateFormat dateFormat, Date validDate) {}
+
+    private static Collection<TestCase> implementations() {
+        return List.of(
+                new TestCase(new DateTimeStringConverter(),
                         Locale.getDefault(Locale.Category.FORMAT), DateFormat.DEFAULT, DateFormat.DEFAULT,
-                        VALID_DATE_WITH_SECONDS, null, null },
+                        null, null, VALID_DATE_WITH_SECONDS),
 
-                { new DateTimeStringConverter(DateFormat.SHORT, DateFormat.SHORT),
+                new TestCase(new DateTimeStringConverter(DateFormat.SHORT, DateFormat.SHORT),
                         Locale.getDefault(Locale.Category.FORMAT), DateFormat.SHORT, DateFormat.SHORT,
-                        VALID_DATE_WITHOUT_SECONDS, null, null },
+                        null, null, VALID_DATE_WITHOUT_SECONDS),
 
-                { new DateTimeStringConverter(Locale.UK),
+                new TestCase(new DateTimeStringConverter(Locale.UK),
                         Locale.UK, DateFormat.DEFAULT, DateFormat.DEFAULT,
-                        VALID_DATE_WITH_SECONDS, null, null },
+                        null, null, VALID_DATE_WITH_SECONDS),
 
-                { new DateTimeStringConverter(Locale.UK, DateFormat.SHORT, DateFormat.SHORT),
+                new TestCase(new DateTimeStringConverter(Locale.UK, DateFormat.SHORT, DateFormat.SHORT),
                         Locale.UK, DateFormat.SHORT, DateFormat.SHORT,
-                        VALID_DATE_WITHOUT_SECONDS, null, null },
+                        null, null, VALID_DATE_WITHOUT_SECONDS),
 
-                { new DateTimeStringConverter("dd MM yyyy HH mm ss"),
+                new TestCase(new DateTimeStringConverter("dd MM yyyy HH mm ss"),
                         Locale.getDefault(Locale.Category.FORMAT), DateFormat.DEFAULT, DateFormat.DEFAULT,
-                        VALID_DATE_WITH_SECONDS, "dd MM yyyy HH mm ss", null },
+                        "dd MM yyyy HH mm ss", null, VALID_DATE_WITH_SECONDS),
 
-                { new DateTimeStringConverter(DateFormat.getDateTimeInstance(DateFormat.LONG, DateFormat.FULL)),
+                new TestCase(new DateTimeStringConverter(DateFormat.getDateTimeInstance(DateFormat.LONG, DateFormat.FULL)),
                         Locale.getDefault(Locale.Category.FORMAT), DateFormat.DEFAULT, DateFormat.DEFAULT,
-                        VALID_DATE_WITH_SECONDS, null, DateFormat.getDateTimeInstance(DateFormat.LONG, DateFormat.FULL) },
-        });
+                        null, DateFormat.getDateTimeInstance(DateFormat.LONG, DateFormat.FULL), VALID_DATE_WITH_SECONDS)
+                );
```

Comment:
This makes it looks like it is the closing bracket related to `new TestCase` for example, while it is from the `List.of`